src/main/java/org/broadinstitute/hellbender/utils/config/ConfigUtils.java

```diff
@@ -0,0 +1,332 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A class to contain configuration utility methods.
+ * Created by jonn on 7/19/17.
+ */
+public class ConfigUtils {
+
+    static private final Logger logger = LogManager.getLogger(ConfigUtils.class);
+
+    // This class just has static methods to help with configuration, so no need for this constructor.
+    private ConfigUtils() {}
```

Comment:
An enum with no members is a better way to express this, as it can't be subclassed and you don't need to declare a dummy constructor. E.g.
```
public enum ConfigUtils {
    ;

    ...
```